src/services/aws/registration/registration.ts

```diff
@@ -33,6 +33,7 @@ const registration_api = aws.injectEndpoints({
         return {
           url: "v1/registration",
           params: { uuid },
+          headers: { authorization: TEMP_JWT },
```

Comment:
@misicnenad , i believe this PR should be pointer to master

Reply:
@ap-justin @misicnenad It's def important enough a fix that we should cherry pick this over to `master` ASAP as well.